server/modules/index.js

```diff
@@ -2,91 +2,91 @@ module.exports = [
     },
     {
         name: 'mongodb.helper',
         dependencies: ['config'],
-        require: __dirname + '/mongodb-helper'
+        module: require(__dirname + '/mongodb-helper')
     },
     {
         name: 'mongodb.repository.user',
         dependencies: ['mongodb.helper'],
-        require: __dirname + '/repositories/user-repository'
+        module: require(__dirname + '/repositories/user-repository')
     },
     {
         name: 'mongodb.repository.project',
         dependencies: ['mongodb.helper'],
-        require: __dirname + '/repositories/project-repository'
+        module: require(__dirname + '/repositories/project-repository')
     },
     {
         name: 'mongodb.repository.buildDefinition',
         dependencies: ['mongodb.helper'],
-        require: __dirname + '/repositories/build-definition-repository'
+        module: require(__dirname + '/repositories/build-definition-repository')
     },
     {
         name: 'mongodb.repository.buildInstance',
         dependencies: ['mongodb.helper'],
-        require: __dirname + '/repositories/build-instance-repository'
+        module: require(__dirname + '/repositories/build-instance-repository')
     },
     {
         name: 'github.apiClient',
         dependencies: ['config'],
-        require: __dirname + '/github-api-client'
+        module: require(__dirname + '/github-api-client')
     },
     {
         name: 'routes.auth.google',
         dependencies: [],
-        require: __dirname + '/routes/auth/google'
+        module: require(__dirname + '/routes/auth/google')
     },
     {
         name: 'instantiation.interpolationHelper',
         dependencies: [],
-        require: __dirname + '/instantiation/interpolation-helper'
+        module: require(__dirname + '/instantiation/interpolation-helper')
     },
     {
         name: 'instantiation.consoleLogger',
         dependencies: [],
-        require: __dirname + '/instantiation/console-logger'
+        module: require(__dirname + '/instantiation/console-logger')
     },
     {
         name: 'instantiation.portProvider',
         dependencies: [],
-        require: __dirname + '/instantiation/port-provider'
+        module: require(__dirname + '/instantiation/port-provider')
     },
     {
         name: 'instantiation.instanceClasses',
         dependencies: ['instantiation.consoleLogger'],
-        require: __dirname + '/instantiation/instance-classes'
+        module: require(__dirname + '/instantiation/instance-classes')
     },
     {
         name: 'instantiation.jobs',
         dependencies: ['config', 'instantiation.portProvider', 'instantiation.interpolationHelper', 'mongodb.repository.buildInstance', 'github.apiClient'],
-        require: __dirname + '/instantiation/jobs'
+        module: require(__dirname + '/instantiation/jobs')
     },
     {
         name: 'instantiation.sourceCodeFetcher',
-        dependencies: ['config', 'instantiation.instanceClasses', 'instantiation.jobs', 'github.apiClient'],
```

Comment:
You're right, we don't need it in this module now. Some of functionalities were extracted to other modules and dependencies were not cleaned up.